src/WrapperRunner/ApplicationForWrapperWorker.php

```diff
@@ -95,6 +95,10 @@ public function runTest(string $testPath): int
             $testSuite     = TestSuite::fromClassReflector($testSuiteRefl);
         }
 
+        EventFacade::emitter()->testSuiteLoaded(
```

Comment:
According to the PHPUnit source code, this call has to be placed between this 2 calls:
https://github.com/paratestphp/paratest/blob/7771e52e192fd9bfd2c3a47139aec57a0c183bd7/src/WrapperRunner/ApplicationForWrapperWorker.php#L222-L224

Reply:
The call requires `$testSuite`, so this would mean that `bootstrap()` would need to be called after `TestSuite::fromClassReflector()`, but `TestSuite::fromClassReflector()` requires the configuration to be bootstrapped. Not sure what the best option here is.

Reply:
I see.

We indeed need the bootstrap to be separated from the test suite loading phase, because in contrary of PHPUnit we load multiple suites per bootstrap.

Let's try your fix and see if we break someone's build :wink: